benefit/data/sheets/CaseEvent/CaseEvent-posthearingb-nonprod.json

```diff
@@ -46,5 +46,31 @@
     "PostConditionState": "dormantAppealState",
     "CallBackURLAboutToSubmitEvent": "${CCD_DEF_SSCS_CCD_ORCHESTRATOR_URL}/send",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "07/07/2023",
+    "CaseTypeID": "Benefit",
+    "ID": "sendToFirstTier",
+    "Name": "Send to First Tier",
+    "Description": "Send to First Tier",
+    "DisplayOrder": 10000,
```

Comment:
need to change display order to put it in alphabetical order

Reply:
Done